recipe/conda_build_config.yaml

```
@@ -541,6 +541,8 @@ occt:
   - 7.4
 openblas:
   - 0.3.*
+openexr:
+  - 2.5
```

Would suggest add `.1` due to the SONAME change issue between 2.5.0 and 2.5.1

```suggestion
openexr:
  - 2.5.1
```